_data/schedules/25-summer.yml

```diff
@@ -10,4 +10,73 @@
     - time: "8:30"
       route_id: gas-works-locks-salmon-bay-loop
     - time: "10:00"
-      route_id: gas-works-lake-union-loop
\ No newline at end of file
+      route_id: gas-works-lake-union-loop
+
+- date: "2025-07-05"
+  plan:
+    - time: "8:30"
+      route_id: westlake-jack-perry-memorial-park-ob
+    - time: "9:30"
+      route_id: westlake-dravus-street-ob
+
+- date: "2025-07-12"
+  plan:
+    - time: "8:30"
+      route_id: locks-discovery-radar-magnolia-discovery-ob
+
+- date: "2025-07-19"
+  plan:
+    - time: "8:30"
+      route_id: northgate-cemeteries-interurban-boeing-creek-ob
+
+- date: "2025-07-26"
+  plan:
+    - time: "8:30"
+      route_id: cse-arb-lwb-seward-loop-cc
+
+- date: "2025-08-02"
+  plan:
+    - time: "8:30"
+      route_id: cap-hill-volunteer-park-westlake
+    - time: "9:10"
+      route_id: westlake-sculpture-marina-ob
+
+- date: "2025-08-09"
+  plan:
+    - time: "8:00"
+      route_id: beacon-i90-luther-burbank-ob
+
+- date: "2025-08-16"
+  plan:
+    - time: "8:30"
+      route_id: lynnwood-jogn-interurban-northgate
+
+- date: "2025-08-23"
+  plan:
+    - time: "8:30"
+      route_id: cid-i90-trail-south-bellvue
+    - time: "9:55"
+      route_id: sobell-follow-2-line-redtech
+
+- date: "2025-08-30"
+  plan:
+    - time: "9:00"
+      route_id: westlake-cranes-alki-sodo
+
+- date: "2025-09-06"
+  plan:
+    - time: "8:30"
+      route_id: cse-woodinville-cafe
+
+- date: "2025-09-13"
+  plan:
+    - time: "8:30"
+      route_id: cse-lwb-frog-pond-i90-trail-sodo
+    - time: "9:05"
+      route_id: sodo-duwamish-trail-1st-loop
+
+- date: "2025-09-20"
```

Comment:
Coast-to-coast is so wonderful and great tradition. Do we want to wait till Sep for this?